### PR TITLE
Upgrade dartdoc

### PR DESCRIPTION
**Pull request by jonasfj**

### CODE REVIEW

File: app/test/frontend/golden/pkg_show_version_page.html [hunk 1]
```diff
@@ -66,7 +66,7 @@ <h1 class="_visuallyhidden">Dart pub</h1>
               </div>
             </div>
           </nav>
-          <div></div>
+          <div>&amp;nbsp;</div>
```
Comment (isoos) [inline app/test/frontend/golden/pkg_show_version_page.html]: this seems weird

File: app/test/frontend/golden/pkg_show_version_page.html [hunk 2]
```diff
@@ -92,7 +92,7 @@ <h2 class="title">foobar_pkg 0.2.0-dev</h2>
 Published 
             <span>Jan 1, 2014</span>
             <!-- &bull; Downloads: X -->
-  • Updated:
+  &amp;bull; Updated:
```
Comment (isoos) [inline app/test/frontend/golden/pkg_show_version_page.html]: also this.

File: app/test/frontend/golden/pkg_show_version_page.html (hunk 1 above)
Comment (jonasfj) [reply app/test/frontend/golden/pkg_show_version_page.html]: Yes, probably because we've upgraded some package.. this one I can't even explain.

Comment (jonasfj) [reply app/test/frontend/golden/pkg_show_version_page.html]: Ah, https://pub.dev/packages/xml#360

Minor change in `package:xml`, fixed it...

**Comment by isoos** [comment]
Nice catch on `package:xml`!

**Pull request merged**